apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -1060,6 +1061,30 @@ export const platform: NavMenuConstant = {
         { name: 'Production Checklist', url: '/guides/platform/going-into-prod' },
       ],
     },
+    {
```

Comment:
Just wonder. In line 1032 you added a line break. If it is needed, please also add a line break here.
```suggestion
    
    {
```

Reply:
Great work! 👍